src/jmclient/wallet.py

```diff
@@ -2153,8 +2156,8 @@ def initialize(cls, storage, network, max_mixdepth=2, timestamp=None,
         if write:
             storage.save()
 
-    def _load_storage(self):
-        super()._load_storage()
+    def _load_storage(self, load_cache=True):
```

Comment:
```suggestion
    def _load_storage(self, load_cache: bool = True) -> None:
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1616/commits/ef1d76e9db09170db94ac0c10efe7b73866b64e3